### PR TITLE
Added the option to use globbing paths on SCSS files

### PR DESCRIPTION
**Pull request by ricardoriogo**
I think that would be useful for styles files organization. I used the gulp-css-globbing plugin, it works with CSS too.
We will use @import 'partials/*'; to import all partials without declare each one.

### CODE REVIEW

File: tasks/shared/Css.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-var gulp   = require('gulp');
-var Elixir = require('../../index');
+var gulp    = require('gulp');
+var cssGlob = require('gulp-css-globbing');
```
Comment (tymondesigns) [inline tasks/shared/Css.js]: this is unnecessary, you should be able to use `$.cssGlobbing()`

File: tasks/shared/Css.js [hunk 2]
```diff
@@ -13,6 +14,7 @@ module.exports = function(options) {
     return (
         gulp
         .src(options.src.path)
+        .pipe(cssGlob({ extensions: ['.css', '.scss', '.sass'] }))
```
Comment (tymondesigns) [inline tasks/shared/Css.js]: not good to hardcode sass file extensions here

File: tasks/shared/Css.js (hunk 1 above)
Comment (ricardoriogo) [inline tasks/shared/Css.js]: Thanks for the tip.

File: tasks/shared/Css.js (hunk 2 above)
Comment (ricardoriogo) [inline tasks/shared/Css.js]: Corrected. Thanks!

**Comment by JeffreyWay** [comment]
Thanks for the PR. 

These sorts of things are tough. I'm not overly against it. Could be useful. But the problem is everyone wants little patches that require additional dependencies. Before you know it, Elixir could end up with dozens and dozens of dependencies. 

My instinct is to close this, unless a bunch of people really want it. If so, I'll merge. But you're the first I've seen bring it up.

**Pull request closed without merging**